imports/plugins/included/product-detail-simple/client/containers/productDetailContainer.js

```diff
@@ -72,8 +72,15 @@ class ProductDetailContainer extends Component {
         });
       } else {
         productId = currentProduct._id;
-        this.setState({ disableAdding: true }); // disable add-to-cart button
         this.setState({ disabledStyle: { cursor: "not-allowed" } }); // sets cursor property style on button
+        this.setState({ disableAdding: true }); // disable add-to-cart button
+
+        // message to user when add-to-cart button gets deactivated
+        Alerts.inline("Add to cart button is currently inactive", "info", {
```

Comment:
No, this is not what I mean. You need to show the button disabled, e.g. show it greyed-out.